File: app/src/main/java/com/spacemonster/webdataviewer/content/data/WebImage.java
package com.spacemonster.webdataviewer.content.data;

/**
 * Created by GE62 on 2017-12-21.
 */

public class WebImage {
    public static final String MAIN_URL = "http://www.gettyimagesgallery.com/";
    public static final String DATA_FILE_PATH = "temp.html";
    public static final String DATA_PAGE_URL = "http://www.gettyimagesgallery.com/collections/archive/slim-aarons.aspx";

    private String imageUrl;

    public WebImage(String imageUrl) {
        this.imageUrl = imageUrl;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }
}

File: app/src/main/kotlin/com/spacemonster/webdataviewer/content/data/WebImage.kt
package com.spacemonster.webdataviewer.content.data

/**
 * Created by GE62 on 2017-12-21.
 */

class WebImage(var imageUrl: String = "") {
    companion object {
        val MAIN_URL = "http://www.gettyimagesgallery.com/"
        val DATA_FILE_PATH = "temp.html"
        val DATA_PAGE_URL = "http://www.gettyimagesgallery.com/collections/archive/slim-aarons.aspx"
    }
}
